packages/checkout/widgets-lib/src/widgets/sale/views/PaymentMethods.tsx

```diff
@@ -90,10 +96,7 @@ export function PaymentMethods() {
           <Link
             sx={{ mx: 'base.spacing.x1' }}
             onClick={
-              () => orchestrationEvents
```

Comment:
thanks @jwhardwick . This will also require updating commerce repo to remove handling of this former event.

Reply:
Thanks @jhesgodi - I'll update Commerce + checkout playground